napari_animation/_qt/dialogs/save_dialog.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+from typing import Any, Callable
+
+from PyQt5.QtWidgets import QFileDialog, QMessageBox
```

Comment:
Can this be a qtpy import instead of PyQt5

Reply:
> @tlambert03, I wonder if you'd ever want to put something in magicgui to assist with open/ save dialogs like this? I didn't see anything in there quite like it right now, but might be something to consider, though it might also be some scope creep too! I can raise an issue on magicgui if you think it warrants further discussion there.

The functionality is there in the [`FileEdit` widget](https://napari.org/magicgui/_autosummary/magicgui.widgets.FileEdit.html#magicgui.widgets.FileEdit), but _just_ opening a dialog should be exposed a little better, and since it's not really a widget, currently requires this nonsense:

```python
from magicgui.application import use_app

show_dialog = use_app().get_obj("show_file_dialog")
file = show_dialog(mode="w")  # returns a string path where they want to save to
```
where "mode" is one of the [`FileDialogModes`](https://napari.org/magicgui/api/types.html#magicgui.types.FileDialogMode)